maro/simulator/scenarios/cim/business_engine.py

```diff
@@ -668,13 +668,6 @@ def _on_action_received(self, event: CascadeEvent):
 
                 action_type: ActionType = getattr(action, "action_type", None)
```

Comment:
As we already assert the action_type is not None is "Action", maybe we can assert the type is "Action" and then action.action_type directly?

Reply:
👌